### ISSUE TITLE
[request] delete after download

### ISSUE THREAD

**Issue by red-dragon** [ipsingh06/seedsync#104]
can you add an option to enable delete after download.

**Comment by TuRDMaN** [comment]
Yes, please! I'm loving this software, but not having this functionality may be a dealbreaker for me if it keeps syncing everything again after the local files are moved/deleted but the remote files are not.

**Comment by dkedinger** [comment]
+1 to this. I was using a command line script to do this on Unraid using lftp but a recent update broke that functionality. SeedSync is great to actually have a GUI to see the status, but would love it to have some option to do the cleanup operations that come with the flags "--delete-excluded" and "--Remove-source-dirs"

**Comment by santakrooz** [comment]
+1 as well. Please! I love this app, works great and xfer is fast. But something like the --remove-source-files option in rsync would enable archives, and file movement workflows that could be fully automated with this option.
